docs/continuous-integration/devcontainers/README.md

```diff
@@ -0,0 +1,74 @@
+# Reusing dev containers within a pipeline
+
+Given a repository with a local development container aka [dev container](../devcontainers/README.md) that contains all the tooling required for development, would it make sense to reuse that container for running the tooling in the Continuous Integration pipelines?
+
+## Options for building devcontainers within pipeline
+
+There are three ways to build devcontainers within pipeline:
+
+- With [GitHub - devcontainers/ci](https://github.com/devcontainers/ci) builds the container with the `devcontainer.json`. Example here: [devcontainers/ci · Getting Started](https://github.com/devcontainers/ci/blob/main/docs/github-action.md#getting-started).
+- With [GitHub - devcontainers/cli](https://github.com/devcontainers/cli), which is the same as the above, but using the underlying CLI directly without tasks.
+- Building the `DockerFile` with `docker build`. This option excludes all configuration/packages specified within the `devcontainer.json`.
```

Comment:
nit: I'd use the term features here since that's a term that VS Code is driving (and matches the term in `devcontainer.json`)

```suggestion
- Building the `DockerFile` with `docker build`. This option excludes all configuration/featuresspecified within the `devcontainer.json`.
```

Reply:
Sure! I'll make a separate commit, because your suggestion is missing a space :)